### PR TITLE
bridge: always set ALLOW_INTERACTIVE_AUTHORIZATION

### PR DESCRIPTION
**Pull request by larskarlitski**

### CODE REVIEW

File: src/bridge/cockpitdbusjson.c [hunk 1]
```diff
@@ -1302,12 +1307,6 @@ array_string_element (JsonArray *array,
   return NULL;
 }
 
-/* Useful analogs for the errors below if not yet defined by glib */
-#if !GLIB_CHECK_VERSION(2,42,0)
-#define G_DBUS_ERROR_UNKNOWN_INTERFACE G_DBUS_ERROR_UNKNOWN_METHOD
-#define G_DBUS_ERROR_UNKNOWN_OBJECT G_DBUS_ERROR_UNKNOWN_METHOD
-#endif
-
```
Comment (stefwalter) [inline src/bridge/cockpitdbusjson.c]: This should actually move to ```cockpitdbuscache.c```

Comment (stefwalter) [inline src/bridge/cockpitdbusjson.c]: Hmmm, no those are already version guarded.

**Comment by martinpitt** [comment]
The first could do with a justification in the commit message -- I thought the cockpit UI already provides a polkit authentication agent, and that does seem to work without the interactive flag. What difference does that flag make?

**Comment by larskarlitski** [comment]
Good point @martinpitt, thanks. I've amended the message.

**Pull request merged**